datasets/lst20/lst20.py

```diff
@@ -0,0 +1,197 @@
+from __future__ import absolute_import, division, print_function
+
+import glob
+import os
+
+import datasets
+
+
+_CITATION = """\
+@article{boonkwan2020annotation,
+  title={The Annotation Guideline of LST20 Corpus},
+  author={Boonkwan, Prachya and Luantangsrisuk, Vorapon and Phaholphinyo, Sitthaa and Kriengket, Kanyanat and Leenoi, Dhanon and Phrombut, Charun and Boriboon, Monthika and Kosawat, Krit and Supnithi, Thepchai},
+  journal={arXiv preprint arXiv:2008.05055},
+  year={2020}
+}
+"""
+
+_DESCRIPTION = """\
+LST20 Corpus is a dataset for Thai language processing developed by National Electronics and Computer Technology Center (NECTEC), Thailand.
+It offers five layers of linguistic annotation: word boundaries, POS tagging, named entities, clause boundaries, and sentence boundaries.
+At a large scale, it consists of 3,164,002 words, 288,020 named entities, 248,181 clauses, and 74,180 sentences, while it is annotated with
+16 distinct POS tags. All 3,745 documents are also annotated with one of 15 news genres. Regarding its sheer size, this dataset is
+considered large enough for developing joint neural models for NLP.
+Manually download at https://aiforthai.in.th/corpus.php
+"""
+
+
+class LST20Config(datasets.BuilderConfig):
+    """BuilderConfig for LST20"""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig for LST20.
+
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(LST20Config, self).__init__(**kwargs)
+
+
+class LST20(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class Lst20(datasets.GeneratorBasedBuilder):
```
This must be the camel case version of the dataset script name `lst20`

you'll need to regenerate the dataset_infos.json after this change